### ISSUE TITLE
Manjaro tray图标显示为eletron通用的图标，而非小飞机图标，局域网代理没有效果

### ISSUE THREAD

**Issue by bickman** [shadowsocksrr/electron-ssr#91]
## Your os

Manjaro Gnome 20.0.2
## The version of electron-ssr

electron-ssr v0.3.0-alpha.5
## What's the problem

1. tray 图标显示的不是小飞机的图标，而是electron的图标
![tray_icon](https://user-images.githubusercontent.com/11661626/83393977-53d0dc80-a42a-11ea-84e7-832d8d9f4471.png)
2. 局域网共享代理没有效果
## App log here

![log](https://user-images.githubusercontent.com/11661626/83395532-0c981b00-a42d-11ea-8285-cab06ebaabb7.png)

**Comment by xVanTuring** [comment]
I've tested the `share over Lan` on my phone, it works. Make sure you are using http proxy (port) instead of socks5, some applications do not support socks5 protocol.

About the icon in the tray, I will try my best to provide a uniform look cross Linux.

**Comment by bickman** [comment]
Yes, `share over lan` only work using http proxy port,it is great, but it is still a little wired that normal 1080 port not working, because both windows and macos ssr works well.

**Comment by bickman** [comment]
By the way, the windows ssr `share over lan` can even use the PAC port which could share pac function for other lan devices.

**Comment by identxxy** [comment]
> By the way, the windows ssr `share over lan` can even use the PAC port which could share pac function for other lan devices.

Is there any way to use `share over lan` with PAC on Linux?
